README.md

```diff
@@ -120,6 +132,8 @@ The RGB gradient goes through gray, while the HSV gradients changes only the hue
 
 ![Gradient Comparison](gfx/readme_gradients.png)
 
+Also note that the center of the gradient crated by interpolating `LinSrgb` values has a somewhat displeasing and dull gray value in the middle. If you wanted to create a gradient as part of a data visualization and as such did not care about physical correctness, this could be improved by using a color space designed to be *perceptually uniform*, like CIE L\*a\*b!
```

Comment:
"crated" -> "created"

Reply:
Also missed a \* after b here and a few other places (saw it while writing the rest of this comment). :slightly_smiling_face:

Oh, and regarding the blandness here, I don't know if L\*a\*b\* would help a lot. I think it's still going to pass through or near some gray or washed out color, just due to red and cyan(ish) being almost complementary colors. They were picked for maximum contrast compared to a hue based interpolation. Not sure if it's the best example to have here, though.